app/about_us/our_values/page.js

```diff
@@ -0,0 +1,67 @@
+import styles from "./page.module.css";
+
+const OurValues = () => {
+  // TASK - React 1 week 1
+  // Create the "Our Values" section
+  // Use the descriptions provided in /app/about_us/README.md
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <>
+      <div className={styles.ourValues}>
+        <p>OUR VALUES</p>
+        <div className={styles.container}>
+          <section className={styles.value}>
+            <h2>01</h2>
+            <h4>Exploration:</h4>
```

Comment:
The heading tag can only be used for headings. Imagine it like headings in books or in newspapers marking the title of the paragraph/subsection/article/etc.
This is violating the accessibility rule of following the heading levels. Levels should always follow each other decreasingly:
h1 > h2 >h3 > h4, and so on. We can not skip any level.

Reply:
If you want to have a visually BIG text, you can always use css:
```
// css:

.valuesNumber {
  font-size: xx-large;
}

// jsx: 
<p className={styles.valuesNumber}>01</p>
<h3>Exploration:</h3>
```

Reply:
More info on headings: https://www.w3.org/WAI/tutorials/page-structure/headings/

Reply:
Ok thanks for providing tutorial for better understanding. It was quite helpful and I made the proper changes accordingly.